src/internal/operators/skip.ts

```diff
@@ -7,6 +7,29 @@ import { OperatorSubscriber } from './OperatorSubscriber';
  * Returns an Observable that skips the first `count` items emitted by the source Observable.
  *
  * ![](skip.png)
+ * 
+ * Skips the values until the sent notifications are equal or less than provided skip count. It raises 
+ * an error if skip count is equal or more than the actual number of emits and source raises an error. An emission
+ * is over when source is reached to its limit.
```

Comment:
Can you delete this last sentence. I get what you are saying, but I think this causes more confusion than it helps the reader